index.js

```diff
@@ -43,7 +43,9 @@ function levelPlugin (fastify, opts, next) {
   fastify.level[name] = levelMore(path || name, opts.options, next)
 }
 
-module.exports = fp(levelPlugin, {
+module.exports = fp(fastifyLeveldb, {
   fastify: '4.x',
   name: '@fastify/leveldb'
 })
+module.exports.default = fastifyLeveldb
```

Comment:
Here the same doubt - I think we should really export the `fp()` one otherwise the function itself does not work with `register`

Reply:
Why? it is actually the same as follow.

```js

module.export = (function(plugin) {
  // it is direct mutation of plugin
  plugin[symbol] = metadata
  return plugin
})(fastifyLeveldb)

module.export.default = fastifyLeveldb // here is mutated function
```

Reply:
Thats imho wrong. Fastify-plugin mutates the function and sets the corresponding symbols. 

https://github.com/fastify/fastify-plugin/blob/b7c1190e72373879c7f14563dff67e6f7f03de96/plugin.js#L45